packages/millicast-sdk/src/View.js

```diff
@@ -53,11 +53,9 @@ export default class View extends BaseWebRTC {
     this.tracksMidValues = {}
     // mapping media ID of RTCRtcTransceiver to DRM Options
     this.drmOptionsMap = null
-    if (mediaElement) {
-      this.on(webRTCEvents.track, e => {
-        mediaElement.srcObject = e.streams[0]
-      })
-    }
+    // cache of events to coordinate re-emitting
+    this.eventQueue = []
+    this.isMainStreamActive = false
```

Comment:
Sorry couldn't figure out how to do this on GitHub around line 49, so sharing here. 
```suggestion
    this.isMainStreamActive = false;
        if (mediaElement) {
      logger.warn('The mediaElement property has been deprecated. In a future release, this will be removed. Please do not rely on this value. Instead, do this in either the `track` or the `active` broadcast event.')
    }
```

Reply:
Done